bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/SIDM_LO/BsTo2DpTo2Mu2e/templates/BsTo2DpTo2Mu2e_run_card.dat

```diff
@@ -146,15 +146,15 @@
 # Minimum and maximum DeltaR distance                                *
 #*********************************************************************
  0.4 = drjj    ! min distance between jets 
- 0.0   = drbb    ! min distance between b's 
- 0.4 = drll    ! min distance between leptons 
+ 0.0   = drbb  ! min distance between b's 
+ 0.0 = drll    ! min distance between leptons 
```

Comment:
I think you can set it to a very small number rather than 0.0 to avoid some integration issue, i.e., 0.01